src/main/java/io/appium/java_client/android/Activity.java

```diff
@@ -0,0 +1,191 @@
+package io.appium.java_client.android;
+
+/**
+ * This is a simple POJO class to support the {@link StartsActivity}.
+ */
+public class Activity {
```

Comment:
I this this POJO should have 2 mandatory parameteres: package and activity. Please add the constructor with these parameters.